components/mpas-ocean/cime_config/buildnml

```diff
@@ -979,8 +979,38 @@ def buildnml(case, caseroot, compname):
                 lines.append('    <var name="waveToOceanMeridionalWindStress"/>')
                 lines.append('    <var name="waveNetOceanZonalWindStress"/>')
                 lines.append('    <var name="waveNetOceanMeridionalWindStress"/>')
+                lines.append('    <var name="langmuirNumber"/>')
                 lines.append('</stream>')
                 lines.append('')
+                lines.append('<stream name="timeSeriesStatsDailyWaveOutput"')
+                lines.append('        type="output"')
+                lines.append('        precision="single"')
+                lines.append('        io_type="pnetcdf"')
```

Comment:
from a discussion with @sbrus89 suggest changing the io_type to`io_type="{}"'.format(ocn_pio_typename)`